mindmeld/auto_annotator.py

```diff
@@ -85,7 +87,7 @@ def _get_file_entities_map(self, action: AnnotatorAction, config):
             rules = config[AnnotatorAction.UNANNOTATE.value]
 
         for rule in rules:
-            pattern = Annotator._get_pattern(rule)
+            pattern = _get_pattern(rule)
```

Comment:
Wont this fail?

Reply:
Nvm looks like its being imported. If so, remove the leading camelcase since its no longer a private function